t/t6026-merge-attr.sh

```diff
@@ -180,4 +180,28 @@ test_expect_success 'up-to-date merge without common ancestor' '
 	)
 '
 
+cat >./custom-merge <<\EOF
+#$SHELL_PATH
+sleep 1 &
+EOF
+chmod +x ./custom-merge
+
+test_expect_success 'custom merge backend with delay' '
+	git reset --hard anchor &&
+	echo "* merge=union" >.gitattributes &&
+	echo "text merge=custom" >>.gitattributes &&
+
+	git config --replace-all \
+	merge.union.driver "true" &&
+	git config --replace-all \
+	merge.union.name "always ours" &&
```

Comment:
Needed for the other files, see the "custom merge backend" tests.
(I did however removed other unnecessary lines.)